tests/core/test_training.py

```diff
@@ -69,10 +69,8 @@ async def test_training_script_without_max_history_set(
     agent = Agent.load(tmpdir)
     for policy in agent.policy_ensemble.policies:
         if hasattr(policy.featurizer, "max_history"):
-            if type(policy) == FormPolicy:
-                assert policy.featurizer.max_history == 2
-            elif type(policy) == MemoizationPolicy:
-                assert policy.featurizer.max_history == OLD_DEFAULT_MAX_HISTORY
+            if type(policy) == MemoizationPolicy:
+                assert policy.featurizer.max_history == DEFAULT_MAX_HISTORY
```

Comment:
I think we might need a more sophisticated check since the value is `None` (I don't think that `== None` works, only `is None`?).

Reply:
Perhaps sth like:
```python
if DEFAULT_MAX_HISTORY is None:
    assert policy.featurizer.max_history is DEFAULT_MAX_HISTORY
else:
    assert policy.featurizer.max_history == DEFAULT_MAX_HISTORY
```

Reply:
Maybe I'm missing something but I think `==` is fine here:

```shell
Python 3.7.10 (default, Feb 26 2021, 10:16:00)
[Clang 10.0.0 ] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> None == None
True
>>> None is None
True
>>> None == 0
False
>>> None == -1
False
```

Reply:
Okay, I've done some more reading and now agree with the `==` 🙂